discord/embeds.py

```diff
@@ -152,15 +106,53 @@ class EmbedFooter(EmbedProxy):
     def __init__(
         self,
         text: str,
-        icon_url: MaybeEmpty[str] = EmptyEmbed,
-        proxy_icon_url: MaybeEmpty[str] = EmptyEmbed,
+        icon_url: str | None = None,
     ) -> None:
-        layer = {
-            k: v
-            for k, v in locals().items()
-            if k in {"text", "icon_url", "proxy_icon_url"} and v is not EmptyEmbed
+        self.text: str = text
+        self.icon_url: str | None = icon_url
+        self.proxy_icon_url: str | None = None
```

Comment:
Same as comment for `EmbedAuthor.__init__`; please reimplement checks for `None`.

Reply:
see https://github.com/Pycord-Development/pycord/pull/2063#discussion_r1190304216